src/plots/linePlot.ts

```diff
@@ -138,6 +139,29 @@ export module Plots {
       return this;
     }
 
+    /**
+     * Gets if croppedRendering is enabled
+     *
+     * When croppedRendering is enabled, lines that will not be visible in the viewport will not be drawn.
+     * Thus if only part of the data is in the viewport, then this option will boost of rendering.
+     * However if all the data will render, enabling will cause an insignificant overhead.
```

Comment:
I'll give this idea to Docs to see if we want to tighten the wording.  I just trimmed what was there from the previous pull request